### PR TITLE
Make version 6 change

### PR DESCRIPTION
**Pull request by henryzbw**

### CODE REVIEW

File: src/scenes/CreateEvent/store.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const messages = defineMessages({
 
 const MAX_LEN_EVENTNAME_HEX = 640;
 const MAX_LEN_RESULT_HEX = 64;
-const TIME_DELAY_FROM_NOW_SEC = 15 * 60;
+const TIME_DELAY_FROM_NOW_SEC = 0;
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: this shouldnt be needed anymore since we dont have a range?

File: src/scenes/CreateEvent/store.js [hunk 2]
```diff
@@ -299,7 +290,7 @@ export default class CreateEventStore {
       this.prediction.startTime = nowPlus(TIME_DELAY_FROM_NOW_SEC);
       this.prediction.endTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + TIME_GAP_MIN_SEC);
       this.resultSetting.startTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + TIME_GAP_MIN_SEC);
-      this.resultSetting.endTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + (TIME_GAP_MIN_SEC * 2));
+      this.resultSetting.endTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + (TIME_GAP_MIN_SEC * 3));
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: the default right now is 48 hours so shouldnt it be times 2?

File: src/scenes/CreateEvent/store.js (hunk 1 above)
Comment (henryzbw) [reply src/scenes/CreateEvent/store.js]: ok, removed.

File: src/scenes/CreateEvent/store.js (hunk 2 above)
Comment (henryzbw) [reply src/scenes/CreateEvent/store.js]: but starttime has one `TIME_GAP_MIN_SEC` from prediction endtime.

**Comment by henryzbw** [comment]
> Also, I will deploy the contract today so you can add the contract metadata for version 6.
> 
> Depends on:
> https://github.com/bodhiproject/bodhi-server/pull/240/files

I'm not sure about the deployBlock.

File: src/scenes/CreateEvent/store.js (hunk 2 above)
Comment (dwalintukan) [reply src/scenes/CreateEvent/store.js]: Ah I see this is for the preset values. Maybe more explicit to set this.prediction.endTime + timeGap *2

**Comment by dwalintukan** [comment]
Don’t worry about the block numbers or metadata yet. We will do a deployment during Thursday’s meeting.

**Comment by dwalintukan** [comment]
@henryzbw lets merge all the ver6 PRs in server + UI during the meeting.

**Pull request merged**